packages/color-brewery/color-brewery.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Christophe Troestler <Christophe.Troestler@umons.ac.be>"
+authors: [ "Christophe Troestler" ]
+license: "LGPL-3 with OCaml linking exception"
+homepage: "https://github.com/Chris00/color_brewery"
+dev-repo: "git+https://github.com/Chris00/color_brewery.git"
+bug-reports: "https://github.com/Chris00/color_brewery/issues"
+doc: "https://Chris00.github.io/color_brewery/doc"
+build: [
+  ["dune" "subst"] {pinned}
+  ["sh" "-c" "ulimit -s 16384 && dune build -p %{name}% -j %{jobs}%"
+   ] {os = "linux"}
+  ["dune" "build" "-p" name "-j" jobs] {os != "linux"}
+]
+depends: [
+  "ocaml" {>= "4.03"}
+  "gg"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.3"}
```
According to the `dune-project` file, this package requires at least dune 1.3

Reply:
It does not really (just was generated by that version of Dune).  Will upload a new tarball saying `1.0`.